runtime/compiler/env/J9ClassEnv.cpp

```diff
@@ -220,6 +223,63 @@ J9::ClassEnv::isAnonymousClass(TR::Compilation *comp, TR_OpaqueClassBlock *clazz
    return comp->fej9()->isAnonymousClass(clazz);
    }
 
+const TR::TypeLayout*
+J9::ClassEnv::enumerateFields(TR::Region& region, TR_OpaqueClassBlock * opaqueClazz, TR::Compilation *comp)
```

Comment:
The region parameter seems to unused in here and is hidden by a local declaration further down. Do we need to pass in a region because this is allocating from the compilation region or should this be allocating using the supplied region to give the caller flexibility in where the data structure is stored?

Reply:
Yes, I think we should use the region that's being passed in here. I have updated this.